archivebox/index.py

```diff
@@ -186,7 +186,8 @@ def patch_links_index(link, out_dir=OUTPUT_DIR):
 
     # Patch HTML index
     html_path = os.path.join(out_dir, 'index.html')
-    html = open(html_path, 'r').read().split('\n')
+    with open(html_path, 'r') as html_file:
+        html = [line[:-1] for line in html_file]
```

Comment:
```suggestion
        html = html_file.read().splitlines()
```

Handles the case where the last line doesn't have a newline instead of chopping off the last character.